rpcs3/util/fixed_typemap.hpp

```diff
@@ -86,6 +89,16 @@ namespace stx
 				std::launder(static_cast<T*>(ptr))->~T();
 			}
 
+			template <typename T>
+			static void call_stop(void* ptr, bool join) noexcept
```

Comment:
maybe use thread_state as a second argument, this way you don't need to redeclare enum values like this.